### ISSUE TITLE
Menu items displayed vertically in Cassiopeia on small screens

### ISSUE THREAD

**Issue by pabloarias** [joomla/joomla-cms#39645]
### Steps to reproduce the issue

It happens in resolutions between 768 and 991 px.

Creating a menu module and assign it to "menu" position.

### Expected result

Menu items displayed horizontally until menu converts in a "burger" icon.

### Actual result

Menu items are displayed vertically. Not horizontally, nor in a "burger".

### System information (as much as possible)

Joomla 4.2.6 and Cassiopeia template. Tested in Firefox and Chrome.

**Comment by brianteeman** [comment]
What is the layout being used for the menu module

![image](https://user-images.githubusercontent.com/1296369/212687992-85d9bd9b-0beb-495b-8685-50753d8fcf62.png)

**Comment by chmst** [comment]
Duplicate #37926.

We had already several questions in the german forum for that. It seems that the breakpoint for metismenu was changedfrom -lg to -md  here https://github.com/joomla/joomla-cms/pull/36900

**Comment by brianteeman** [comment]
Thats not the problem as it also applies to the non metis collapsible menu which has always been -md

It should expand at md and above

The problem is that between -md and -lg it is being displayed as vertical.

So going from a small to large size it does the following
- burger
- vertical
- horizontal

It should  not go to vertical

**Comment by pabloarias** [comment]
@brianteeman, the layout being used for the menu module is "Collapsible Dropdown".

But the problem is with all layouts: "Default", "Dropdown" and "Collapsible Default Menu" too.

@chmst, yes, this issue seems to be duplicated with #37926.

I'm using this CSS code to solve it in my user.css file:

```css
@media (min-width: 768px) {
    .container-header .mod-menu {
        display: flex;
        flex-direction: row;
    }
}
```<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39645">issues.joomla.org/tracker/joomla-cms/39645</a>.</sub>

**Comment by AKASH-PRASAD7** [comment]
I want to work on this issue, please assign me.

**Comment by alikon** [comment]
there is no need to assign the issue, simply submit a pull request

**Comment by Mohammadkhan18** [comment]
> @brianteeman, the layout being used for the menu module is "Collapsible Dropdown".
> 
> But the problem is with all layouts: "Default", "Dropdown" and "Collapsible Default Menu" too.
> 
> @chmst, yes, this issue seems to be duplicated with #37926.
> 
> I'm using this CSS code to solve it in my user.css file:
> 
> ```css
> @media (min-width: 768px) {
>     .container-header .mod-menu {
>         display: flex;
>         flex-direction: row;
>     }
> }
> ```
> 
> This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/39645](https://issues.joomla.org/tracker/joomla-cms/39645).

so is this the class where we have to work?

**Comment by pabloarias** [comment]
Ok, this bug is solved in #40574.

Thank you!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39645">issues.joomla.org/tracker/joomla-cms/39645</a>.</sub>

**Comment by Quy** [comment]
Thanks for the update.
